Source/WebKit/UIProcess/Cocoa/WebPreferencesCocoa.mm

```diff
@@ -130,6 +130,17 @@ static void setDebugBoolValueIfInUserDefaults(const String& identifier, const St
     store.setBoolValueForKey(key, [object boolValue]);
 }
 
+static void setDebugUInt32ValueIfInUserDefaults(const String& identifier, const String& keyPrefix, const String& globalDebugKeyPrefix, const String& key, WebPreferencesStore& store)
+{
+    id object = debugUserDefaultsValue(identifier, keyPrefix, globalDebugKeyPrefix, key);
+    if (!object)
+        return;
+    if (![object respondsToSelector:@selector(unsignedIntegerValue)])
+        return;
+
+    store.setUInt32ValueForKey(key, [object unsignedIntegerValue]);
+}
+
```

Comment:
Thanks for bringing this back, confirmed the implementation matches what I removed in 9def6e6f0258ddd9171d32bec2fffb684dbc7b8b